.github/workflows/release.yml

```
@@ -0,0 +1,154 @@
+  # Run every Sunday at 12:00 UTC
+  schedule:
+    - cron: '0 12 * * 0'
+
+env:
+  CARGO_TERM_COLOR: always
+  ASSET_DIR: ${{ github.workspace }}/frontend/dist
+  SQLX_OFFLINE: "true"
+
+jobs:
+  build:
+    name: Build
+    strategy:
+      matrix:
+        target: 
+          - os: ubuntu-latest
+            name: x86_64-unknown-linux-musl
+            binary: backend/target/x86_64-unknown-linux-musl/release/abacus
+          - os: macos-latest
+            name: aarch64-apple-darwin
+            binary: backend/target/aarch64-apple-darwin/release/abacus
+          - os: windows-latest
+            name: x86_64-pc-windows-msvc
+            binary: backend/target/x86_64-pc-windows-msvc/release/abacus.exe
+    runs-on: ${{ matrix.target.os }}
+    defaults:
+      run:
+        working-directory: ./backend
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install dependencies
+        run: npm ci
+        working-directory: ./frontend
+      - name: Build
+        run: npm run build
+        working-directory: ./frontend
+      - name: Setup Rust
+        run: rustup update stable && rustup default stable
+      - name: Install target
+        run: rustup target add ${{ matrix.target.name }}
+      - name: Install musl
+        run: sudo apt-get update && sudo apt-get install -y musl-tools
+        if: matrix.target.os == 'ubuntu-latest'
+      - name: Cargo cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cargo/.crates.toml
+            ~/.cargo/.crates2.json
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            ./backend/target/
+          key: ${{ runner.os }}-release-cargo-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: ${{ runner.os }}-release-cargo-
+      - name: Run tests
+        run: cargo --locked test --release
+      - name: Build
+        run: cargo --locked build --release --features memory-serve --target=${{ matrix.target.name }}
+      - uses: actions/upload-artifact@v4
+        with:
+          name: abacus-${{ matrix.target.os }}
+          path: ${{ matrix.target.binary }}
+
+  playwright-d2d:
+    name: Playwright d2d tests (${{ matrix.os }}, ${{ matrix.shardIndex }}/${{ matrix.shardTotal }})
+    needs:
+      - build
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        shardIndex: [1, 2, 3]
+        shardTotal: [3]
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        working-directory: frontend
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install dependencies
+        run: npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
```

```suggestion
        run: npx playwright install --with-deps --no-shell
```

because of #632 (am almost ready to submit the PR)